CONTRIBUTING.md

```diff
@@ -106,34 +141,62 @@ There is two different goals:
-```bash
-uv run odoo -d oca-custom -i oca_all --stop-after-init --without-demo=
-```
-
-- run tests using pytest launcher
+Run tests using pytest launcher.
 
 ```bash
 uv run pytest --odoo-database oca-custom --cov ./oca_psc_team/ oca_psc_team/
 ```
 
-### Update OCB Branch
+#### Update OCB Branch
 
 ```bash
 uv sync -P odoo
 ```
 
-### Update a specific OCA module dependency using the latest pypi release
+#### Update a specific OCA module dependency using the latest pypi release
 
 ```bash
 uv sync -P odoo14-addon-<module-name>
 ```
 
-### Bump all dependencies to the latest version
+#### Bump all dependencies to the latest version
 
 ```bash
 uv sync -U
 ```
+
+## FAQ
+
+#### How can I start contributing in OCA toolings?
+* Get to know the manifest document of OCA's Internal Tools team: [Scope & Objective](https://docs.google.com/document/d/1RcOUstPJDev1bgPZcNXWiHBt5PeqraisU5bKIZatcnY/edit?pli=1&tab=t.0#heading=h.jrsgv4k2u2ao).
+* Write to us at [internaltools@odoo-community.org](mailto:internaltools@odoo-community.org).
+* Install a test environment on your machine of the OCA's Odoo instance by following the [§ Getting Started](#getting-started).
+
+#### How to communicate with the OCA Internal Tools?
+Our main communication channel is the mailing list [internaltools@odoo-community.org](mailto:internaltools@odoo-community.org).
+For task-related discussion, also directly use the Chatter of the Odoo tasks.
+
+#### Where is the tasks backlog of the OCA Internal Tools?
+We use the Odoo project **OCA internal tools workgroup** on the OCA's Odoo instance to organize our tasks and priorities.
+A public access to the project can be provided to OCA members who contributes to the tooling tasks (not requesting privacy form signing).
+
+#### How to access the backend of OCA's Odoo instance?
+It can be useful to access Odoo back-end for both task management and browse instance's modules and data. For such, prerquisites are:
+* being a member of the OCA and the **OCA Internal Tools group**
+* be registered on OCA's Odoo instance
+* fullfil and send back the **Data protection & privacy** form, available on OCA website [Resources / How to guides / Protect data & privacy when you support OCA projects](https://odoo-community.org/privacy)
+
+#### What are useful Github repositories?
+* The current **oca-custom** is the main repository. It contains both all Odoo modules dependencies of OCA instance in `oca_all/__manifest__.py` and all configuration to build an Odoo test instance the `uv`, as described in [Getting Started](#getting-started).
+* [**apps-store**](https://github.com/OCA/apps-store/tree/14.0) holds mechanisms of OCA modules replication to official Odoo's App Store
+
+#### How to get representive data, for troubleshooting and test?
+Contact the mailing list to get a neutralized and anonymized database.
+
+#### How to gain command line access and read logs?
+Only a few people have admin server access, please reach the mailing list for further details.
```

Comment:
Command-line access to the 'odoo' user on the server. But this is also something you can get when you sign the privacy form.